modules/tinymce/src/core/test/ts/browser/bookmark/BookmarksTest.ts

```diff
@@ -223,4 +223,53 @@ describe('browser.tinymce.core.bookmark.BookmarksTest', () => {
     assertApproxRawContent(editor, '<p>abc</p>');
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
   }));
+
+  context('Get bookmark should work if the first element of the content is a comment', () => {
+    const outsideButton: SugarElement<HTMLButtonElement> = SugarElement.fromHtml('<button id="getBookmarkButton">Get Bookmark</button>');
+    const setupElement = () => {
+      const element = SugarElement.fromTag('textarea');
+      Insert.append(SugarBody.body(), outsideButton);
+
+      return {
+        element,
+        teardown: () => Remove.remove(outsideButton)
+      };
+    };
+
+    const hook = TinyHooks.bddSetupFromElement<Editor>({
+      base_url: '/project/tinymce/js/tinymce',
+      menubar: false,
+      toolbar: false,
+      statusbar: false,
+    }, setupElement, []);
+
+    const setSelectionToCommentAndTriggerGetBookmark = (editor: Editor, content: string) => {
```

Comment:
This name still hasn't been fixed.

Reply:
changed in `setContentAndTriggerGetBookmark`